src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
![image](https://user-images.githubusercontent.com/117547096/208889512-0a8e362e-ec9d-468d-b315-76e14f7645c4.png)
```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
        List<String> dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writetoFile(report, toFileName);
    }
```